bitmap.go

```diff
@@ -53,18 +53,17 @@ func (r *Renderer) drawBitmap(g shaping.Glyph, bitmap api.GlyphBitmap, img draw.
 			}
 		}
 
-		scaled := resize.Resize(uint(fixed266ToFloat(g.Width)*r.PixScale), uint(fixed266ToFloat(g.Height)*r.PixScale),
-			sub, resize.NearestNeighbor)
-		draw.Draw(img, scaled.Bounds().Add(image.Point{X: int(x), Y: int(y)}), scaled, image.Point{}, draw.Over)
+		rect := image.Rect(int(x), int(y), int(fixed266ToFloat(g.Width)*r.PixScale), int(fixed266ToFloat(g.Height)*r.PixScale))
+		scale.NearestNeighbor.Scale(img, rect, sub, sub.Bounds(), draw.Over, nil)
 	case api.JPG, api.PNG, api.TIFF:
 		pix, _, err := image.Decode(bytes.NewReader(bitmap.Data))
 		if err != nil {
 			return err
 		}
 
 		h := r.FontSize * r.PixScale
-		scaled := resize.Resize(uint(h), uint(h), pix, resize.Bicubic)
```

Comment:
It still looks like you're forcing the destination to be square. Is that an oversight? Also, should the max values be something like `int(w+s), int(h+y)` to offset them properly relative to the origin coordinates?

Reply:
Thanks for catching that I missed it in the coordinate updates, resolved now